### ISSUE TITLE
PropMap should

### ISSUE THREAD

**Issue by orbifx** [Kakadu/lablqt#24]
https://github.com/Kakadu/lablqt/blob/master/lablqml/propMap_stubs.c#L53

> QObject::connect returns a connection object that can be used to
> disconnect reliably later on. You should return that to ocaml in case
> the user wants to disconnect a ocaml closure later on.

-- Goswin

**Comment by Kakadu** [comment]
I really don't think that it is critical.

**Comment by orbifx** [comment]
Yes, it's not for applications whose property maps live for the whole time the application runs.

It's a note for future reference when the implementation needs to be completed.

**Comment by Kakadu** [comment]
Completed? LOL
